internal/xds/translator/testdata/in/ratelimit-config/masked-remote-address-match.yaml

```diff
@@ -0,0 +1,22 @@
+name: "first-listener"
+address: "0.0.0.0"
+port: 10080
+hostnames:
+- "*"
+routes:
+- name: "first-route"
+  rateLimit:
+    global:
+      rules:
+      - cidrMatch:
+          cidr: "192.168.0.0/16"
+          ipv6: false
+          ones: 16
```

Comment:
ones ?

Reply:
come from:
```
// Size returns the number of leading ones and total bits in the mask.
// If the mask is not in the canonical form--ones followed by zeros--then
// Size returns 0, 0.
func (m IPMask) Size() (ones, bits int) {
	ones, bits = simpleMaskLength(m), len(m)*8
	if ones == -1 {
		return 0, 0
	}
	return
}
```

Reply:
maybe change it to `maskLen`?

Reply:
yeah, isn't this the struct
```
type CIDRMatch struct {
 	CIDR    string
 	IPv6    bool
 	MaskLen int
 }
 ```
 so the input should be `maskLen`
 
 rateLimitConfig doesnt use `maskLen` so this test is passing :)

Reply:
yeah, I found it.